src/docs/development/packages-and-plugins/developing-packages.md

```diff
@@ -223,6 +223,11 @@ environment:
 To create a plugin package, use the `--template=plugin`
 flag with `flutter create`.
 
+Starting from flutter 1.20.0:
+Use the `--platforms` option to specify all the platforms that the plugin
+supports. Available platforms are: android, ios, web, linux, macos and windos.
+If no platforms are specified, a plugin project supports no platforms will be created.
```

Comment:
=> If no platforms are specified, the resulting project doesn't support any platforms.

Really? This seems like a bug.  :)